### PR TITLE
Feature/merge 126.0.2 tag

### PR DESCRIPTION
**Pull request by eg-ayoub**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## Release 126.0.2 (in development)
```
Comment (TeddyAndrieux) [inline CHANGELOG.md:3]: If you may also add the new `## Release 126.0.3 (in development)` :)

**Review by TeddyAndrieux — APPROVED** [review]

**Comment by eg-ayoub** [comment]
ping

**Pull request merged**